.gitignore

```diff
@@ -1,42 +1,45 @@
-#Autosave files
-*~
-
-#build
-[Oo]bj/
-[Bb]in/
-packages/
-TestResults/
-
-# globs
-Makefile.in
-*.DS_Store
-*.sln.cache
-*.suo
-*.cache
-*.pidb
-*.userprefs
-*.usertasks
-config.log
-config.make
-config.status
-aclocal.m4
-install-sh
-autom4te.cache/
-*.user
-*.tar.gz
-tarballs/
-test-results/
-Thumbs.db
-
-#Mac bundle stuff
-*.dmg
-*.app
-
-#resharper
-*_Resharper.*
-*.Resharper
-
-#dotCover
-*.dotCover
-
-.idea/
+#Autosave files
+*~
+
+#build
+[Oo]bj/
+[Bb]in/
+packages/
+TestResults/
+
+# globs
+Makefile.in
+*.DS_Store
+*.sln.cache
+*.suo
+*.cache
+*.pidb
+*.userprefs
+*.usertasks
+config.log
+config.make
+config.status
+aclocal.m4
+install-sh
+autom4te.cache/
+*.user
+*.tar.gz
+tarballs/
+test-results/
+Thumbs.db
+
+#Mac bundle stuff
+*.dmg
+*.app
+
+#resharper
+*_Resharper.*
+*.Resharper
+
+#dotCover
+*.dotCover
+
+# Vim swap files
+*.swp
```

Comment:
This is the only change - the rest is because I changed from windows line endings to unix line endings. I can revert that if you want (though that would be inconsistent with the rest, which all use unix line endings).